README.md

```diff
@@ -50,14 +50,17 @@ formView.delegate = self
 4. Set a FormViewModel to your formView
 
 
-A `FormViewModel` can be any class that implements the `FormViewModel` delegate. 
+A `FormViewModel` can be any class that implements the `FormViewModel` delegate.
 For a class to implement `FormViewModel` delegate  you only need to define an array of `FormItem`.
-Each `FormItem` will determine the behavior and validation of each text field in your form. 
-`FormItem` can be a text field, two text fields in line or a "section header"
-More about fields configuration in the `FormItem` section.
+Each `FormItem` will determine the behavior and validation of each text field in your form.
+Out of the box, RSFormView provides the following subclasses of `FormItem`:
+- `TextCellItem`: A single line of text, or a "section header"
```

Comment:
"a single line of text" may be misleading since the label actually break lines, so I'd go with: a single label or "section header"